docs/server.md

```diff
@@ -30,7 +32,7 @@ Add substrate url and account's mnemonics in config.json before running the serv
 ## Running
 
 ```bash
-yarn run
+yarn run run
```

Comment:
maybe we can change the name [here](https://github.com/threefoldtech/twin_server_v2/blob/development/package.json#L28) from `run` to `server` to make the command `yarn run server`

Reply:
done